app/src/main/java/com/mobile/tandil/javabaseproject/mvp/presenter/ParkingPresenter.java

```diff
@@ -11,5 +11,9 @@ public ParkingPresenter(ParkingContract.Model model, ParkingContract.View view)
         this.model = model;
         this.view = view;
     }
-}
 
+    @Override
+    public void onShowButtonPressed() {
```

Comment:
here you should set the value to the model, so in the future you already have all the structure done.

1-. set the value to the model
2-. show the value in the view, getting it from the model

Reply:
done